iFixit/react-commerce
# Add collection tagline

This PR closes #102 

## Changelog

- Adds optional tagline
![image](https://user-images.githubusercontent.com/4640135/138049657-82df765a-b102-477a-bfb1-422d5a4b3a02.png)


## QA

1. Visit the Vercel PR preview
2. Visit the parts collection page
3. Verify that the tagline is rendered according to data from Strapi
